lib/ProductOpener/Display.pm

```diff
@@ -6519,7 +6519,11 @@ sub display_field($$) {
 
 
 	if ((defined $value) and ($value ne '')) {
-		if (($field eq 'link') and ($value =~ /^http/)) {
+		# See https://stackoverflow.com/a/3809435
+		if (($field eq 'link') and ($value =~ /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&\/\/=]*)/)) {
+			if ($value !~ /https?:\/\//) {
+				$value = 'http://' + $value;
```

Comment:
in Perl + will change the left and right operands to numbers, this will result in $value = 0

```suggestion
				$value = 'http://' . $value;
```

Reply:
Oh yeah, too used to Java XD